README.md

```diff
@@ -84,6 +86,33 @@ All of the specified resources in the IAM policy do not have to exist prior to t
 }
 ```
 
+## Terraform Deployment
+
+Here is an example of how you would use the module in your own terraform deployment:
+
+```text
+module "github-actions-runner-aws" {
+  source = "github.com/cmsgov/github-actions-runner-aws?ref=v1.0.0"
+
+  # ECR variables
+  container_name          = "github-runner"
+  allowed_read_principals = local.principals_ro_access
+  ci_user_arn             = data.aws_iam_user.github_ci_user.arn
+
+  # ECS variables
+  environment               = "dev"
+  ecs_desired_count         = 0
+  ecs_vpc_id                = data.aws_vpc.example_east_sandbox.id
```

Comment:
Can you include example of how the VPC is being looked up as a data source?

Reply:
I ended up making this usage more generic, but did add a section on how to use data sources to look-up info in AWS